src/Workspaces/Core/Portable/Notification/AbstractGlobalOperationNotificationService.cs

```diff
@@ -15,7 +15,6 @@ internal abstract class AbstractGlobalOperationNotificationService : IGlobalOper
 
         public abstract GlobalOperationRegistration Start(string reason);
 
-        public abstract void Cancel(GlobalOperationRegistration registration);
```

Comment:
we had an entire pipeline used to report if the global operation was canceled or completed.  this effectively just passed around a boolean through the pipeline.  but nothing ever read that boolean.  so i removed it all.